Fix for flipped tables

Bunfus opened the pull request:
# About the pull request
This PR adds new table subtypes for flipped tables. This makes it easier for mappers to view/place flipped tables on maps and also fixes an old issue where mapped-in flipped tables still behaved as regular tables (Fixes #7602).

# Explain why it's good for the game
Having less bugs is good.

# Testing Photographs and Procedure
<details>
<summary>Screenshots</summary>
Flipped tables are now easier to see in SDMM
<img width="854" height="858" alt="test" src="https://github.com/user-attachments/assets/6c9cf011-f736-4a9a-bdf0-b86c1442706c" />
</details>

# Changelog
:cl: Nemesis1337
fix: fixed mapped-in flipped tables to function as expected
spellcheck: fixed a few minor typos
code: added flipped table subtypes
imagetweak: fixed wooden barricade sprite
maptweak: replaced all var edited tables with new subtypes
/:cl:

Bunfus committed:
Fixed flipped tables and corrected some examine typos

Bunfus committed:
Fixed flipped tables and corrected some examine typos

Bunfus committed:
One line fix

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Reinforced table tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Extra fixes

Bunfus committed:
Fixup maps in TGM format

066ceb592003: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Merge branch 'master' into barricade-changes

Bunfus committed:
Merge branch 'master' into barricade-changes

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

Bunfus committed:
Merge branch 'master' of https://github.com/cmss13-devs/cmss13 into barricade-changes

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Merge branch 'cmss13-devs:master' into barricade-changes

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into barricade-changes

Bunfus committed:
Merge branch 'master' into barricade-changes

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into barricade-changes

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into barricade-changes

Bunfus committed:
Multi-z fixes

Bunfus committed:
Merge branch 'master' into barricade-changes

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into barricade-changes

Bunfus committed:
Merge branch 'barricade-changes' of https://github.com/Bunfus/cmss13 into barricade-changes

Bunfus committed:
Merge branch 'master' into barricade-changes

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into barricade-changes

Bunfus committed:
Merge branch 'barricade-changes' of https://github.com/Bunfus/cmss13 into barricade-changes

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into barricade-changes

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into barricade-changes

Bunfus committed:
Minor tile adjustments

Bunfus committed:
Small typo fixes

Bunfus committed:
Fixup maps in TGM format

6e4265317940: maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm

Automatically commited by: tools\mapmerge2\fixup.py

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into barricade-changes

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

Bunfus committed:
Added flipped tables to the Tyrargo and new Varadero inserts

Nanu308 reviewed (dismissed):
mapping seems fine, just name replacements and grammar mostly

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into barricade-changes

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into barricade-changes

Bunfus committed:
Merge branch 'master' into barricade-changes

Drulikar commented on code/game/objects/structures/tables_racks.dm:
```diff
@@ -299,7 +306,7 @@
 		SPAN_NOTICE("You start disassembling [src]."))
 		playsound(src.loc, 'sound/items/Ratchet.ogg', 25, 1)
-		if(do_after(user, 50 * user.get_skill_duration_multiplier(SKILL_CONSTRUCTION), INTERRUPT_ALL|BEHAVIOR_IMMOBILE, BUSY_ICON_BUILD, src))
+		if(do_after(user, 50 * user.get_skill_duration_multiplier(SKILL_CONSTRUCTION), INTERRUPT_ALL|BEHAVIOR_IMMOBILE, BUSY_ICON_BUILD))
```
Why are you removing the `target` argument? Doing so would mean that this do_after will then proceed if say two people disassembled the table because the target is no longer checked.

Drulikar reviewed (changes requested):
This PR repaths things in maps, but does not have an update paths script. See https://github.com/cmss13-devs/cmss13/blob/master/.github/CONTRIBUTING.md#a-note-on-prs-altering-maps

fira commented:
Hi, closing for inactivity - if you want to push through with these changes please address Drathek's request for migration scripts (you can ask for help on Discord) and open a new PR. Thank you!
